logo/README.md

```diff
@@ -0,0 +1,26 @@
+## Official logo of Redux-recompose  
+[![supported by](https://img.shields.io/badge/Supported%20by-Wolox.💗-blue.svg)](https://www.wolox.com.ar/) 
+
+Our development team and designers have selected our new official logo of `redux-recompose`.
+
+### Vertical logo
+
+![Vertical Logo Redux-recompose](./images/Redux_vertical_small@2x.png)
+
+Download as small [PNG](./images/Redux_vertical_small@2x.png)
+
+Download as big [PNG](./images/Redux_vertical_big@2x.png)
+
+### Horizontal logo
+
+![Horizontal Logo Redux-recompose](./images/Redux_Horizontal_small@2x.png)
+Download as small [PNG](./images/Redux_Horizontal_small@2x.png)
+
+Download as big [PNG](./images/Redux_Horizontal_small@2x.png)
+
+
+## Credits
+
+The Redux-recompose logo was designed by [Jorge Montoya](https://www.behance.net/icon_studio).  
+Thanks to [Manuel Battan](https://github.com/mvbattan), [Mariano Zicavo](https://github.com/marianozicavo), [Ana Ospina](https://github.com/Anisospina) and [the community](https://github.com/Wolox/redux-recompose) for contributing and give a little love to project 🥰.
```

Comment:
![image](https://user-images.githubusercontent.com/7256996/54564784-6be64880-49ab-11e9-866f-52d0de6349a1.png)
It looks like something is not showing correctly